src/com/horev/Color.java

```diff
@@ -0,0 +1,10 @@
+package com.horev;
+
+/**
+ * This enum represents a color of player
+ * @author Aviad Bagno
+ */
+enum Color {
+    COLOR_WHITE,
```

Comment:
lets just have it as WHITE or BLACK

Reply:
I think that it will be more convenient in the code to know that it is a color
But why did you approve that pull request if we have not finished discussing it yet?